CMSgov/design-system
# [WNMGDS-531] Add autofocus and ref to <Alert>

### Add
- add `alertRef` prop to <Alert> div block
- add `autoFocus` prop to autofocus on the <Alert> div block on initial loading of the alert
- support other undefined props to the Alert component
 
The props perform similar function as 'inputRef' & 'focusTrigger' on other components

## How to test
1. Run the doc site locally 
1. Add the `alertRef={(elem) => console.log(elem)}` prop to the `<Alert>` example 
1. Add the `autoFocus` prop to the `<Alert>` example 
1. See that alert code block is printed on console and focus is activated on mount of Alert page

Demo website: http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-531/Add-autofocus-and-ref-to-alert/

## NOTE
Checking with Josh Nelson to determine if there's a case for `alertRef` prop or would the `autoFocus` to focus on first mount be sufficient?
